### ISSUE TITLE
`prefer-called-with` should report `toHaveBeenCalledOnce()`

### ISSUE THREAD

**Issue by nami8824** [vitest-dev/eslint-plugin-vitest#910]
## Summary

`prefer-called-with` should report `toHaveBeenCalledOnce()` and autofix it to `toHaveBeenCalledExactlyOnceWith()`.

`prefer-called-with` currently reports call assertions that do not check arguments, such as:

```js
expect(fn).toHaveBeenCalled()
expect(fn).toBeCalled()
```

and autofixes them to:

```js
expect(fn).toHaveBeenCalledWith()
expect(fn).toBeCalledWith()
```

However, `toHaveBeenCalledOnce()` is not reported, even though it also only checks the call count and does not verify the call arguments.

 I am not fully sure whether this should be autofixable, but `prefer-called-with` already autofixes `toHaveBeenCalled()` to  `toHaveBeenCalledWith()`, so autofixing `toHaveBeenCalledOnce()` to  `toHaveBeenCalledExactlyOnceWith()` would be consistent with the current rule behavior.

## Background

`toHaveBeenCalledOnce()` was previously requested in #173 and briefly supported in #184, but it was later removed in #231. At the time, there was no valid `*OnceWith` matcher to autofix to.

Vitest now provides `toHaveBeenCalledExactlyOnceWith()`, which is the correct target for this case.

## Proposal

Report:

```js
expect(fn).toHaveBeenCalledOnce()
```

and autofix it to:

```js
expect(fn).toHaveBeenCalledExactlyOnceWith()
```

## Examples

Invalid:

```js
expect(fn).toHaveBeenCalledOnce()
```

Valid:

```js
expect(fn).toHaveBeenCalledExactlyOnceWith()
```

**Comment by veritem** [comment]
fixed in #911
